pyproject.toml

```diff
@@ -0,0 +1,87 @@
+[build-system]
+requires = ["hatchling>=0.18.0"]
+build-backend = "hatchling.build"
+
+[project]
+name = "in-toto"
+description = "A framework to define and secure the integrity of software supply chains"
+readme = "README.md"
+license = "Apache-2.0"
+requires-python = "~=3.7"
+authors = [
+    { name = "New York University: Secure Systems Lab", email = "in-toto-dev@googlegroups.com" },
+]
+maintainers = [
+    { name = "Justin Cappos", email = "jcappos@nyu.edu" },
+    { name = "Santiago Torres", email = "santiago@nyu.edu" },
```

Comment:
maybe this would be a good time to update my email :)

Reply:
```suggestion
    { name = "Santiago Torres", email = "santiagotorres@purdue.edu" },
```